drivers/staging/android/ion/ion_dummy_driver.c

```diff
@@ -24,13 +24,18 @@
 #include <linux/io.h>
 #include "ion.h"
 #include "ion_priv.h"
+#include "ion_unmaped_heap.h"
 
 static struct ion_device *idev;
 static struct ion_heap **heaps;
 
 static void *carveout_ptr;
 static void *chunk_ptr;
 
+#ifdef CONFIG_ION_DUMMY_UNMAPED_HEAP
```

Comment:
From hear on to a search and replace from "unmaped" to "unmapped", I won't add more comments about it further down.